pages/agent/v3/cli_lock.md

```diff
@@ -0,0 +1,26 @@
+# `buildkite-agent lock`
+
+The Buildkite Agent's `lock` subcommands provide the ability to coordinate multiple concurrent builds on the same host that access shared resources.
+
+With the agent-api experiment enabled, processes can acquire and release a lock with `acquire` and `release` subcommands. For the special case of performing setup once for the life of the agent (and waiting until it is complete) there are the `do` and `done` subcommands. These provide an alternative to using `flock` or OS-dependent locking mechanisms. To enable the agent-api experiment, include the `--experiment=agent-api` flag when starting the agents.
```

Comment:
I think it would be good to more obviously call out the experiment here and separate that from the feature info. Thinking something similar to the [Experimental features](https://buildkite.com/docs/agent/v3#experimental-features) section:

```suggestion
>🛠 Experimental feature
> The agent-api experiment must be enabled to use the `lock` command. To enable the agent-api experiment, include the `--experiment=agent-api` flag when starting the agent. 

With the `lock` command, processes can acquire and release a lock using the `acquire` and `release` subcommands. For the special case of performing setup once for the life of the agent (and waiting until it is complete), there are the `do` and `done` subcommands. These provide an alternative to using `flock` or OS-dependent locking mechanisms.
```

What do you think?